evalml/automl/automl_search.py

```diff
@@ -955,7 +955,7 @@ def search(self, show_iteration_plot=True):
             full_rankings = self.full_rankings
             current_batch_idx = full_rankings["id"].isin(new_pipeline_ids)
             current_batch_pipeline_scores = full_rankings[current_batch_idx][
-                "mean_cv_score"
+                "validation_score"
```

Comment:
Since `validation_score` now will either hold the validation score of a training-validation split or the mean cv score of the CV folds, it will be the better measure to use for sorting model performance

Reply:
Curious... are there any performance implications to this? Are different pipelines now ranked differently, potentially resulting in different scores? Might be over the top but could be good to have perf test results just to see changes to rankings, if any :)

Reply:
@angela97lin Pipelines shouldn't be ranked differently! `validation_score` now will hold what `mean_cv_score` previously held, which would be the the mean score in CV folds and the validation score in TV splits. The main difference, now, is that `mean_cv_score` will be `NaN` when there are no CV folds, which means that we need to use `validation_score` to sort the pipelines

Reply:
I see, thank you for explaining, I understand better now 👍